README.md

```diff
@@ -1 +1,158 @@
-https://hub.docker.com/r/microsoft/oms/
+# Trying the container solution pack for Microsoft Operations Management Suite
+
+## What is the Microsoft Operations Management Suite?
+
+The Microsoft Operations Management Suite (OMS) is a software-as-a-service offering from Microsoft that allows Enterprise IT to manage any hybrid cloud. It offers log analytics, automation, backup and recovery, and security and compliance.  Sign up for a free account at [http://mms.microsoft.com](http://mms.microsoft.com) or read more about here: [https://www.microsoft.com/en-us/server-cloud/operations-management-suite/overview.aspx](https://www.microsoft.com/en-us/server-cloud/operations-management-suite/overview.aspx).
```

Comment:
Same as above. In general, I like markdown files or license files to be limited to 70 bytes or so per line, when possible (not always possible of the line contains a very long URL or something).

And again, not worth changing just for this. Just keep it in mind in the future.